### ISSUE TITLE
Request: get Azure DevOps integration webhook endpoint via data source

### ISSUE THREAD

**Issue by coolbrewed** [spacelift-io/terraform-provider-spacelift#469]
Hey team! Would it be possible to be able to retrieve the Azure DevOps integration webhook endpoint via data source in Terraform?

The [Azure DevOps integration](https://registry.terraform.io/providers/spacelift-io/spacelift/latest/docs/data-sources/azure_devops_integration) is already able to get the other key info needed for [adding a webhook subscription to an Azure DevOps Project](https://docs.spacelift.io/integrations/source-control/azure-devops#configuring-webhooks) to support auto-deployment, but it looks like the webhook endpoint isn't part of that query bundle yet.

# Use case
I'm working to help onboard my organization with a goal to be able to create delegated admin Spaces via Terraform. Each Space would also have an Azure DevOps Project created for it, with an initial Repo associated with their starter administrative Stack, and the set of auto-deployment webhooks created for the Project.

Currently, there's just two components we're waiting on/watching for to be able to manage the resources fully via Terraform:
1. The Azure DevOps Provider to [add support for a resource for webhook servicehooks](https://github.com/microsoft/terraform-provider-azuredevops/pull/461)
2. The Spacelift.io Provider to add support for fetching the Azure DevOps integration webhook endpoint

# Priority
Definitely low!

This would mainly be a nice-to-have: we do have a manual work-around, and the ideal solution does require another Provider to resolve one of the other blockers. 

However, if we could at least retrieve all the webhook info from Spacelift via Terraform, then we could use that info to create the webhooks programmatically/via API until the Azure DevOps Provider adds the option. So it'd still help make the automation a bit easier.

(Thank you in advance for consideration, it's much appreciated!)
